modules/python/python_loader.cmake

```diff
@@ -48,6 +48,9 @@ if(NOT OpenCV_FOUND)  # Ignore "standalone" builds of Python bindings
   else()
     list(APPEND CMAKE_PYTHON_BINARIES_PATH "'${LIBRARY_OUTPUT_PATH}'")
   endif()
+  if (DEFINED CUDA_TOOLKIT_ROOT_DIR)
+    list(APPEND CMAKE_PYTHON_BINARIES_PATH "'${CUDA_TOOLKIT_ROOT_DIR}/bin'")
```

Comment:
1. It is CUDA specific, so it should be guarded by `if(HAVE_CUDA)`
2. It is Windows only, so add `if(WIN32)` guard too
3. It is unlikely that we want path from the "build" machine. Perhaps python loader should reuse target machine environment. Something like:
```
os.environ.get('SOME_CUDA_ROOT_DIR_ENV_PARAMETER', `${CUDA_TOOLKIT_ROOT_DIR}`) + '/bin'
```